libs/blocks/modal/modal.css

```diff
@@ -166,3 +166,11 @@
     max-width: calc((100% - 6px) - 2em);
   } 
 }
+
+@media (max-width: 1200px) {
+  .dialog-modal {
+    width:100%;
+    max-width:100%;
+    max-height:100%;
```

Comment:
I'm not sure we want to expand all modals to these dimensions on viewports widths lower than `1200px`. Also, as a nit, you should leave a space between property and value for better readability.

Reply:
This could be an alternative solution.

/*parent selector alternative for dialog-modal*/
.dialog-modal:has(.fragment[data-path*="fragments/trial-modals"]){
     width: 100%;
    max-width: 100%;
    max-height: 100%;
}

Reply:
Hi @sharath-kannan as discussed please add this css to only commerce-frame css class and not to the dialog-modal class

Reply:
changed this to a much specific selector.